packages/hoppscotch-common/package.json

```diff
@@ -67,6 +67,7 @@
     "jsonpath-plus": "^7.0.0",
     "lodash-es": "^4.17.21",
     "lossless-json": "^2.0.8",
+    "minisearch": "^6.1.0",
```

Comment:
Since `minisearch` is added, the `fuse.js` dependency should be removed.

Reply:
If `fuse.js` is used elsewhere, then it's okay.

Reply:
It is still used actually, the Keyboard Shortcuts flyout still uses Fuse. We can port it over if that is required.

Reply:
It would be appropriate to port that over to minisearch. Two deps to do the same thing doesn't make sense.

Reply:
Cool porting over. I didn't do that because it wasn't in the scope of the PR.

Reply:
Okay, done! Please check the new commits